app/templates/etc/prod/conf.sh

```diff
@@ -0,0 +1,28 @@
+
+# Define overrides for init script defaults here, defaults shown in comments..
+
+# name of the project
+#NAME=${0##*/}
+
+# location of your project's root
+# SOURCE_DIR=/space/projects/$NAME.live
+
+# js entry point
+#SOURCE_FILE=boot.js
+
+# server script to be launched
+#SERVER_SCRIPT=./node-server/node-app.js
+
+# port at which node server should be listening
+#APP_PORT=9040
+APP_PORT=80
```

Comment:
I just checked all of my POC demo boxes.  All of them have httpd installed, and all of them have the service off for each runlevel.  Here is an example:
[bholt@energy ~]$ chkconfig --list httpd
httpd          	0:off	1:off	2:off	3:off	4:off	5:off	6:off